cmd/runhcs/container.go

```diff
@@ -349,13 +349,14 @@ func createContainer(cfg *containerConfig) (_ *container, err error) {
 		}
 
 		// Determine the network namespace to use.
-		if cfg.Spec.Windows.Network != nil && cfg.Spec.Windows.Network.NetworkSharedContainerName != "" {
-			err = stateKey.Get(cfg.Spec.Windows.Network.NetworkSharedContainerName, keyNetNS, &netNS)
-			if err != nil {
-				if _, ok := err.(*regstate.NoStateError); !ok {
-					return nil, err
-				}
-			}
+		if cfg.Spec.Windows.Network != nil && cfg.Spec.Windows.Network.NetworkNamespace != "" {
+			netNS = cfg.Spec.Windows.Network.NetworkNamespace
+			//err = stateKey.Get(cfg.Spec.Windows.Network.NetworkNamespace, keyNetNS, &netNS)
```

Comment:
Had to comment this out, since this call was failing to get the value from registry

Reply:
Again. This is to support the RS4 Argon workflows. We need to leave this code alone. Just change it to add the conditon for the new code.

```go
		// Determine the network namespace to use.
		if cfg.Spec.Windows.Network != nil {
			if cfg.Spec.Windows.Network.NetworkNamespace != "" {
				netNS = cfg.Spec.Windows.Network.NetworkNamespace
			} else if cfg.Spec.Windows.Network.NetworkSharedContainerName != "" {
				err = stateKey.Get(cfg.Spec.Windows.Network.NetworkSharedContainerName, keyNetNS, &netNS)
				if err != nil {
					if _, ok := err.(*regstate.NoStateError); !ok {
						return nil, err
					}
				}
			}
		}
```

Reply:
Sure